exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillStatementImpl.java

```diff
@@ -159,24 +230,25 @@ public void cleanUp() {
   public int getQueryTimeout() throws AlreadyClosedSqlException
   {
     throwIfClosed();
-    return 0;  // (No no timeout.)
```

Comment:
Interestingly, AvaticaStatement returns the timeout value that was set... but does not honour it! :) 
Originally the setter would trigger a NotSupported exception and the explicit return was the default 0
Now that we're able to support it, I can read Avatica's value directly.